server/compute.js

```diff
@@ -143,6 +143,11 @@ module.exports = function compute({ changes = {}, user, segments, events = [] },
     if (d.kind === "N" || d.kind === "E") {
       _.set(memo, d.path, d.rhs);
     }
+    // when we have an array updated we set the whole
+    // array in `changed` constant
+    if (d.kind === "A") {
+      _.set(memo, d.path, _.get(updatedUser, d.path, []));
```

Comment:
@michaloo what if we have this : 

- previous value is string -> new value is array
- previous value is array -> new value is string

are those cases handled ?